debian/qubes-core-agent.install

```diff
@@ -100,6 +101,7 @@ usr/bin/qvm-move-to-vm
 usr/bin/qvm-open-in-dvm
 usr/bin/qvm-open-in-vm
 usr/bin/qvm-run-vm
+usr/bin/qvm-terminal
```

Comment:
I wonder if `qvm-console` wouldn't be a better name. There is `xl console`, `virsh console`, etc.
Then, same for `admin.vm.Terminal` -> `admin.vm.Console`.